docs/Overview.md

```diff
@@ -143,11 +143,11 @@ burn the funds on L2, allowing the user to reclaim them through the `finalizeEth
 L2 -> L1 communication, in contrast to L1 -> L2 communication, is based only on transferring the information, and not on
 the transaction execution on L1.
 
-From the L2 side, there is a special zkEVM opcode that saves `l2ToL1Log` in the L2 block. A validator will send all
```

Comment:
Please update document to have description of what is `batch` and what is `block`